src/pages/patientView/PatientViewGenePanelModal/GenesList.tsx

```diff
@@ -0,0 +1,140 @@
+        }
+        return genes;
+    };
+
+    genesDividedToColumns = (genes: GenePanelToGene[]) => {
+        return chunk(genes, this.genesCountPerColumn(genes.length));
+    };
+
+    renderColumns = () => {
+        const filtered = this.filterGenes();
+        if (filtered.length === 0) {
+            return <td>No matches.</td>;
+        }
+        return this.genesDividedToColumns(filtered).map(col => (
+            <td>
+                {col.map(each => (
+                    <p key={each.entrezGeneId}>{each.hugoGeneSymbol}</p>
+                ))}
+            </td>
+        ));
+    };
+
+    getDownloadData = () => {
+        const downloadData = [['Genes'], ...this.genesDividedToRows()];
+        return serializeData(downloadData);
+    };
+
+    genesDividedToRows = () => {
+        const { genes } = this.props.genePanelCache;
+        const genesByColumns = this.genesDividedToColumns(genes);
+        const genesByRows = [];
+        for (let i = 0; i < this.genesCountPerColumn(genes.length); i++) {
+            const genesPerRow = [];
+            for (let j = 0; j < this.columnsCount(); j++) {
+                genesPerRow.push(
+                    genesByColumns[j] && genesByColumns[j][i]
+                        ? genesByColumns[j][i].hugoGeneSymbol
+                        : ''
+                );
+            }
+            genesByRows.push(genesPerRow);
+        }
+        return genesByRows;
+    };
+
+    columnsCount = () => {
+        return this.props.columns || 1;
+    };
+
+    genesCountPerColumn = (totalLength: number) => {
+        return Math.ceil(totalLength / this.columnsCount());
+    };
+
+    render() {
+        return (
+            <div id={this.props.id}>
+                <h4 className="modal-title panel-name">
+                    {this.props.panelName}
+                </h4>
+                <span>
+                    Number of genes: {this.props.genePanelCache.genes.length}
+                </span>
+                <div className="pull-right has-feedback input-group-sm">
+                    <input
+                        type="text"
+                        value={this.state.filter}
+                        onInput={(e: React.ChangeEvent<HTMLInputElement>) =>
+                            this.handleChangeInput(e.target.value)
+                        }
+                        className="form-control"
+                    />
+                    <span
+                        className="fa fa-search form-control-feedback"
+                        aria-hidden="true"
+                    />
+                </div>
+                <SimpleCopyDownloadControls
+                    className="pull-right genes-list-controls"
+                    downloadData={this.getDownloadData}
+                    downloadFilename={`gene_panel_${this.props.panelName}.tsv`}
+                    controlsStyle="BUTTON"
+                    containerId={this.props.id}
+                />
+                <table>
```

Comment:
you should use bootstrap styles for this table.

Reply:
I have switched to using the shared `SimpleTable`.